vivo-project/VIVO
# [ i18n sprint ] move qr codes about page translations to n3 configurations

[VIVO-languages companion PR](https://github.com/vivo-project/VIVO-languages/pull/113)
# What does this pull request do?
This pull request is one of many efforts to improve language neutral Freemarker templates and thus eliminate the need to support language specific templates

Removed AboutQrCodesController
Changed about qr code page url from /qrcode/about to /about_qrcode
# How to test # 
1. Create a person
2. Click on qr code image, larger image will pop up, click on large image to get to Export qr code page.
3. Click on the link "What is this?"
4. You should be on about qr codes page. Try changing locales to verify it is available in multiple languages
